--- rpm-sequoia-1.7.0/Cargo.toml~	2024-07-09 10:47:06.000000000 +0200
+++ rpm-sequoia-1.7.0/Cargo.toml	2025-02-06 10:02:49.753329864 +0100
@@ -43,7 +43,7 @@
 
 # We explicitly do not want to enable Sequoia's decompression support.
 # Hence we only select a crypto backend.
-default = ["crypto-nettle"]
+default = ["crypto-openssl"]
 crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
 crypto-rust = ["sequoia-openpgp/crypto-rust"]
 crypto-cng = ["sequoia-openpgp/crypto-cng"]
